draft/ZEP0001.md

```diff
@@ -421,82 +389,87 @@ Related work for sharding (motivation for storage transformers):
 
 ## Implementation
 
-The following projects contains the implementation of Zarr Version 3 protocol:
+The following projects contain implementations of Zarr Version 3 protocol:
 
-- zarrita: https://github.com/alimanfoo/zarrita
+- zarrita: <https://github.com/alimanfoo/zarrita>
```

Comment:
None of these contain an implementation of zarr v3 at the moment, due to the substantial spec changes.  I think zarr-python's v3 implementation is more than slightly outdated.

I assume zarrita won't be getting updated.

I think xtensor-zarr is also an abandoned project.

So perhaps just say that zarr-python support for v3 is in progress with a link to the issue tracker.  We can add other implementations later when they exist.

Reply:
Fair enough, I updated this :+1: